app/src/main/res/values/strings.xml

```diff
@@ -46,10 +46,10 @@
 
     <string name="firstrun_dialog_message">
         DNSSetter lets you change your DNS servers. This is temporary and it will not resist if your
-        IP address changes or if you reboot your device.\n
+        IP address changes or if you reboot your operating system.\n
```

Comment:
I think that *reboot* is more of a hardware term and is more commonly used with *device* instead of *operating system*, but I may be mistaken.
If you really insist, why not use "restart your operating system"?